packages/flutter/test/material/text_field_test.dart

```diff
@@ -14,6 +14,7 @@ import 'package:flutter/services.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart' show DragStartBehavior;
 
+import '../rendering/mock_canvas.dart';
```

Comment:
This is a bit unexpected. You imported a file but didn't change anything else?

Reply:
actually this probably should be here, but add a test in this file that checks for the drawRRect called for Material TextField on iOS